instrumentation/jaxws/jaxws-3.0-cxf-4.0/javaagent/src/test/groovy/TestWsServlet.groovy

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import hello.HelloServiceImpl
+import io.opentelemetry.api.trace.Span
+import io.opentelemetry.context.Context
+import io.opentelemetry.instrumentation.api.instrumenter.LocalRootSpan
+import org.apache.cxf.jaxws.EndpointImpl
+import org.apache.cxf.message.Message
+import org.apache.cxf.phase.AbstractPhaseInterceptor
+import org.apache.cxf.phase.Phase
+import org.apache.cxf.transport.servlet.CXFNonSpringServlet
+
+import jakarta.servlet.ServletConfig
+
+class TestWsServlet extends CXFNonSpringServlet {
```

Comment:
we have been attempting to convert all of our groovy tests to java (see #7195), is there a reason these tests need to be in groovy, or could you write them in java instead?

Reply:
Since these tests are copy pasted from older cxf tests it is in my opinion fine to keep them in groovy.